### PR TITLE
[#158812641] Fix ButtonHamburger doesn't show close icon

### PR DESCRIPTION
**Pull request by fikrikarim**
pivotaltracker.com/story/show/158812641

## What happened
- Fix ButtonHamburger close button
- Add space to introduction paragraph
 
## Insight
- Because I moved the ButtonHamburger out of a `.styled-select__toggle` div, it doesn't have a `z-index: 100` property, which is required so the ButtonHamburger is still shown when the list shown.
- The styling for ButtonHamburger should be included in ButtonHamburger stylesheet, not in ProjectFilter stylesheet as previously done.

## Proof Of Work
screenflow

### CODE REVIEW

File: src/containers/AppLayout.jsx [hunk 1]
```diff
@@ -22,11 +22,23 @@ class AppLayout extends React.Component {
...
+              </span>{" "}
+              We build world-class web and mobile applications by leveraging
+              open source technology and software. We also contribute by open
+              sourcing and releasing every day the libraries, tools and
+              applications our team creates.{" "}
+              <span className="d-none d-md-inline">
+                Our software focuses on Ruby, Javascript, Swift and Kotlin. But
+                we also venture into other areas such as Augmented Reality,
+                Machine Learning and Blockchain.
+              </span>
```
Comment (olivierobert) [inline src/containers/AppLayout.jsx]: Please stop running prettier on file not affected by this PR. This formatting is quite ridiculous 😭

File: src/assets/stylesheets/components/_btn-hamburger.scss [hunk 2]
```diff
@@ -6,6 +6,8 @@
   padding-top: 0.25rem;
   padding-left: 0px;
   padding-right: 0px;
+  z-index: 100;
```
Comment (olivierobert) [inline src/assets/stylesheets/components/_btn-hamburger.scss]: Prefer using the SCSS variables `$zIndex` (in `_variables.scss`) to reduce the entropy of z-index values:

```
z-index: map-get($zIndex, higher);
```

File: src/containers/AppLayout.jsx (hunk 1 above)
Comment (fikrikarim) [reply src/containers/AppLayout.jsx]: @olivierobert alright

File: src/assets/stylesheets/components/_btn-hamburger.scss (hunk 2 above)
Comment (fikrikarim) [reply src/assets/stylesheets/components/_btn-hamburger.scss]: Alrightt, I'm not used to using scss (or any CSS cool things 😭 ) but hopefully I'll used to it soon.

**Pull request merged**